---
layout: cv
title: Graham's CV
---
# Graham D Rose
PhD educated bioinformatics scientist with over ten year's experience in high-throughput sequencing from doctoral training, research, and government level positions, including Imperial College Healthcare NHS Trust, Wellcome Trust Sanger Institute and Public Health England, providing a wide analytical and applied skill set in the genomics space, coupled to a strong molecular biology training.

<div id="webaddress">
<i class="fa fa-envelope"></i> <a href="mailto:grahamdrose@gmail.com">graham</a>
|
<i class="fa fa-linkedin"></i> <a href="https://uk.linkedin.com/in/graham-rose-033a9829">LinkedIn</a>
|
<i class="fa fa-github"></i> <a href="https://byronmews.github.io">byronmews.github.io</a>
</div>


## Current

I am currently based within the GOSH Bioinformatics group at [Great Ormond Street Hospital for Children NHS Foundation Trust](http://www.labs.gosh.nhs.uk/laboratory-services/genetics), which is the lead centre for the North Thames Genomic Labortory Hub. Initially funded through [Genomics England](https://www.genomicsengland.co.uk/the-100000-genomes-project/), this now permanent role involves software development, analysis and management skill sets for the high-throughput sequencing data generated as part of routine diagnostic service, and previously the 100,000 genomes project.

## Employment

`09.2019-present`
__Great Ormond Street Hospital__ Permanent bioinformatician within the diagnostic bioinfomatics team, combining the development and maintenance of a suite of bespoke web based applications and analytical pipelines, and to facilitate Clinical Scientists and researchers within interogating genomic datasets.

`04.2019-03.2021`
__Springer Nature__ Freelance Scientific Editor, taking on the developmental editing of research articles and reports.

`04.2019-09.2019`
__Great Ormond Street Hospital__ Contract bioinformatician role to help develop and maintain a software management tool for handling clinical data retrieved from Genomics England as part of the 100,000 genomes project. As well as wider integration across the Bioinformatics group software projects.

`10.2018-04.2019`
__Great Ormond Street Hospital__ Honorary bioinformatician role involving a collaboration to build on a bioinformatics management tool for handling clinical data retrieved from Genomics England as part of the 100,000 genomes project.

`05.2018-present`
__Imperial College London__ Honorary research fellow within the National Heart Lung Institute, providing bioinformatics support for several rare disease datasets generated as part of the 100,000 genomes project.

`01.2018-04.2019`
__Imperial College Healthcare NHS Trust__ Bioinformatics scientist working on the 100,000 Genomes Project, and part of the West London Genomic Medicine Centre (GMC).

`04.2016-04.2018`
__Faculty of Genomic Medicine, Queen Mary University London__ Visiting lecturer, responsible for delivering part of the HEE MSc in Genomic Medicine ‘Application of Genomics in Infectious Disease’ module.

`07.2014-01.2018`
__Public Health England__ Computational Biologist role, responsible for building and delivering genomic and metagenomic research and development projects using a range of NGS platforms.

`09.2013-07.2014`
__The Pirbright Institute__ Characterising terabase-scale sequence data from the transcriptome of Anopheles gambiae, as part of the Vector Molecular Biology group

`02.2008-07.2009`
__Wellcome Trust Sanger Institute__ Finishing and comparative analysis within the Pathogen Genomics group, including Clostridium difficile and Mycobacterium
tuberculosis. 

## Education

`09.2009-06.2013`
__MRC National Institute of Medical Research__ Pathogen Genomics PhD (supervisors: Prof. Douglas Young and Prof. Sebastein Gagneux). My thesis was entitled _A genomic and transcriptomic study of lineage-specific variation in Mycobacterium tuberculosis_, available via [UCL Senate House](http://discovery.ucl.ac.uk/1397251/)                                    

`09.2006-08.2007`
__University of Leeds__ MRes Bioinformatics and Computational Biology (_Distinction_)

`09.2003-06.2006`
__University of Leeds__ BSc Microbiology (_Hons_) 2.1

## Presentations

### National

`2017`
Microbiology Society event: Microbiome session at Porton Down, "Challenges of the Unknown"

`2016`
Illumina User General Meeting: Invited scientific talk session, "The preterm infant gut microbiome"


## Publications

<!-- ### Journals -->

`2021`

__The Mycobacterium tuberculosis sRNA F6 regulates expression of groEL/S.__
J Houghton, A Rodgers, __G. Rose__, KB Arnvig. Microbiology spectrum. 2021

__Premature neonatal gut microbial community patterns supporting an epithelial TLR-mediated pathway for necrotizing enterocolitis__
Shaw, A. G., Sim, K., __Rose, G.__, Wooldridge, D. J., Li, M. S., Misra, R. V., Gharbia, S., & Kroll, J. S. BMC microbiology. 2021

`2020`
__The airways microbiome of individuals with asthma treated with high and low doses of inhaled corticosteroids.__
Martin, M. J., Zain, N. M. M., Hearson, G., Rivett, D. W., Koller, G., Wooldridge, D. J., __Rose, G__, Gharbia, S. E., Forbes, B., Bruce, K. D., & Harrison, T. W. PloS one. 2020

`2017`
__Complete Genome Sequences of BK Polyomavirus Strains from Two Patients with Urinary Tract Infection, Sequenced Using the Ion Torrent Platform.__
__G. Rose__, K. Ranellou, R. Misra, C. Crump, D. Wooldridge, S. Parmar, C. Maddren, S. Gharbia, H.  Jalal. Genome Announcements. 2017

__The Challenges of Identifying Mycobacterium to the Species Level using MALDI‐TOF MS__
__G. Rose__, R. Culak, T. Chambers, S. E. Gharbia, H. N. Shah, Y. Kim, JS. Kim.  MALDI‐TOF and Tandem MS for Clinical Microbiology. 2017.

__Antibiotic resistance potential of the healthy preterm infant gut microbiome.__
__G. Rose__, A. G. Shaw, K. Sim, D. J. Wooldridge, M. Li, S. Gharbia, R. Misra, J. S. Kroll. PeerJ. 2017.

`2016`
__Dosage compensation in the African malaria mosquito Anopheles gambiae.__
__G. Rose__, E. Krzywinska, J. Kim, L. Revuelta, L. Ferretti, and J. Krzywinski. Genome Biology and Evolution. 2016.

__Development of a candidate reference material for adventitious virus detection in vaccine and biologicals manufacturing by deep sequencing__. ET Mee, MD Preston, PD Minor, S Schepelmann; __CS533 Study Participants__. Vaccine. 2016.

`2015`
__Genomic diversity of BK polyomavirus in East of England–Preliminary data.__
K Ranellou, C Crump, R Misra, D Wooldridge, __G Rose__, S Gharbia, S Parmar, M Curran, N Verlander, H Zhang, H Jalal. Journal of Clinical Virology. 2015.

__Challenges of the Unknown: Clinical Application of Microbial Metagenomics.__
__G. Rose__, D. J. Wooldridge, C. Anscombe, E. T. Mee, R. V. Misra, and S. Gharbia. International Journal of Genomics. 2015.

__Complete genome sequence of the hypervirulent bacterium Clostridium difficile strain G46, ribotype 027.__
T. Gaulton, R. Misra, __G. Rose__, P. Baybayan, R. Hall, J. Freeman, J. Turton, S. Picton, J. Korlach, S. Gharbia, and H. Shah. Genome Announcements. 2015.

`2013`
__A genomic and transcriptomic study of lineage-specific variation in Mycobacterium tuberculosis.__
__GD. Rose__. University College London. 2013.

__Mapping of Genotype-Phenotype Diversity among Clinical Isolates of Mycobacterium tuberculosis by Sequence-Based Transcriptional Profiling.__
__G. Rose__, T. Cortes, I. Comas, M. Coscolla, S. Gagneux, and D. B. Young. Genome Biology and Evolution. 2013.

__Genome-wide Mapping of Transcriptional Start Sites Defines an Extensive Leaderless Transcriptome in Mycobacterium tuberculosis.__
T. Cortes, O. T. Schubert, __G. Rose__, K. B. Arnvig, I. Comas, R. Aebersold, and D. B. Young Cell Reports. 2013.

__A Small RNA Encoded in the Rv2660c Locus of Mycobacterium tuberculosis is Induced during Starvation and Infection.__
J. Houghton, T. Cortes, O. Schubert, __G. Rose__, A. Rodgers, M. De Ste Croix, R. Aebersold, D. B. Young, and K. B. Arnvig. PLoS ONE. 2013.

__Use of whole genome sequencing to determine the microevolution of Mycobacterium tuberculosis during an outbreak.__
M. Kato-Maeda, C. Ho, B. Passarelli, N. Banaei, J. Grinsdale, L. Flores, J. Anderson, M. Murray, __G. Rose__, L. M. Kawamura, N. Pourmand, M. a. Tariq, S. Gagneux, and P. C. Hopewell. PLoS One. 2013.

`2012`
__The heterogeneous evolution of multidrug-resistant Mycobacterium tuberculosis.__
B. Müller, S. Borrell, __G. Rose__, and S. Gagneux. Trends Genet. 2012.

`2011`
__Whole-genome sequencing of rifampicin-resistant Mycobacterium tuberculosis strains identifies compensatory mutations in RNA polymerase genes.__
I. Comas, S. Borrell, A. Roetzer, __G. Rose__, B. Malla, M. Kato-Maeda, J. Galagan, S. Niemann, and S. Gagneux. Nature Genetics 2011.

__Sequence-based analysis uncovers an abundance of non-coding RNA in the total transcriptome of Mycobacterium tuberculosis.__
K. B. Arnvig, I. Comas, N. R. Thomson, J. Houghton, H. I. Boshoff, N. J. Croucher, __G. Rose__, T. T. Perkins, J. Parkhill, G. Dougan, and D. B. Young. PLoS Pathogens. 2011.

__Genotypic and phenotypic modifications of Neisseria meningitidis after an accidental human passage.__
H. Omer, __G. Rose__, K. a. Jolley, E. Frapy, J. R. Zahar, M. C. J. Maiden, S. D. Bentley, C. R. Tinsley, X. Nassif, and E. Bille. PLoS One. 2011.

`2010`
__Evolutionary dynamics of Clostridium difficile over short and long time scales.__
M. He, M. Sebaihia, T. D. Lawley, R. A. Stabler, L. F. Dawson, M. J. Martin, K. E. Holt, H. M. B. Seth-Smith, M. a Quail, R. Rance, K. Brooks, C. Churcher, D. Harris, S. D. Bentley, C. Burrows, L. Clark, C. Corton, V. Murray, __G. Rose__, S. Thurston, A. van Tonder, D. Walker, B. W. Wren, G. Dougan, and J. Parkhill. PNAS. 2010.

`2009`
__Comparative genome and phenotypic analysis of Clostridium difficile 027 strains provides insight into the evolution of a hypervirulent bacterium.__
R. A. Stabler, M. He, L. Dawson, M. Martin, E. Valiente, C. Corton, T. D. Lawley, M. Sebaihia, M. A. Quail, __G. Rose__, D. N. Gerding, M. Gibert, M. R. Popoff, J. Parkhill, G. Dougan, and B. W. Wren. Genome Biology. 2009.



## Technical skills

* Linux CLI tools and sysadmin of lab/groups VMs and servers
* Shell scripting - bash/awk/sed
* Python - pandas, beautifulsoup, RESTful APIs, microservices
* Full Stack development - Django, JavaScript, Docker, PostgreSQL, and asynchronous tasks with Celery
* HPC - batching, lustre
* Java/Groovy - pipelining with bpipe, snakemake and Nextflow workflows
* Perl - general scripting
* VBA - lifting legacy NHS code into modern languages
* R - bioconductor, DESeq
* GitLab DevOps, setting up maintaining CI/CD pipelines


## Areas of expertise
* Bioinformatics
* Analysis pipeline development
* NGS and variant calling
* Bacterial genomics


## Awards

Public Health England Pump Prime Funding (PPF) round 2017 (viral surveillance study)

MRC scholarship (PhD fees + stipend)

BBSRC postgraduate scholarship (MRes fees + stipend)


## Links

<!-- fa are fontawesome, ai are academicons -->
* <i class="fa fa-envelope"></i> <a href="mailto:grahamdrose@gmail.com">grahamdrose</a><br />
* <i class="fa fa-github"></i> <a href="http://github.com/byronmews">github</a><br />
* <i class="fa fa-linkedin"></i> <a href="https://www.linkedin.com/in/graham-rose-033a9829/">LinkedIn</a>
* <i class="ai ai-google-scholar"></i> <a href="http://scholar.google.com/citations?user=TIkDO_cAAAAJ&hl=en">Google scholar</a>
* <i class="ai ai-orcid"></i> <a href="https://orcid.org/0000-0002-5097-5426">ORCID</a>

<!-- ### Footer

Last updated: Sep 2018 -->
