<!doctype html>
<html>
<head>
  <meta charset=utf-8 />
  <title> Graham's CV</title>
  <link href="media/cv-screen.css" type="text/css" rel="stylesheet" media="screen">
  <link href="media/cv-print.css" type="text/css" rel="stylesheet" media="print">
  <link href="media/academicons.min.css" rel="stylesheet">

  <!-- imports -->
  <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
  <link href='http://fonts.googleapis.com/css?family=Open+Sans:400,300,700' rel='stylesheet' type='text/css'>
</head>
<body>
  <div id="main">
    <div id="content">
    <h1 id="graham-d-rose">Graham D Rose</h1>
<p>PhD educated bioinformatics scientist with over ten year’s experience in high-throughput sequencing from doctoral training, research, and government level positions, including Imperial College Healthcare NHS Trust, Wellcome Trust Sanger Institute and Public Health England, providing a wide analytical and applied skill set in the genomics space, coupled to a strong molecular biology training.</p>

<div id="webaddress">
<i class="fa fa-envelope"></i> <a href="mailto:grahamdrose@gmail.com">graham</a>
|
<i class="fa fa-linkedin"></i> <a href="https://uk.linkedin.com/in/graham-rose-033a9829">LinkedIn</a>
|
<i class="fa fa-github"></i> <a href="https://byronmews.github.io">byronmews.github.io</a>
</div>

<h2 id="current">Current</h2>

<p>I am currently based within the GOSH Bioinformatics group at <a href="https://norththamesgenomics.nhs.uk/about-the-gms/the-glh/">Great Ormond Street Hospital for Children NHS Foundation Trust</a>, which is the lead centre for the North Thames Genomic Labortory Hub. Initially funded through <a href="https://www.genomicsengland.co.uk/the-100000-genomes-project/">Genomics England</a>, this now permanent role involves software development, analysis and management skill sets for the high-throughput sequencing data generated as part of routine diagnostic service, and previously the 100,000 genomes project.</p>

<h2 id="employment">Employment</h2>

<p><code class="highlighter-rouge">09.2019-present</code>
<strong>Great Ormond Street Hospital</strong> Permanent bioinformatician within the diagnostic bioinfomatics team, combining the development and maintenance of a suite of bespoke web based applications and analytical pipelines, and to facilitate Clinical Scientists and researchers within interogating genomic datasets.</p>

<p><code class="highlighter-rouge">04.2019-03.2021</code>
<strong>Springer Nature</strong> Freelance Scientific Editor, taking on the developmental editing of research articles and reports.</p>

<p><code class="highlighter-rouge">04.2019-09.2019</code>
<strong>Great Ormond Street Hospital</strong> Contract bioinformatician role to help develop and maintain a software management tool for handling clinical data retrieved from Genomics England as part of the 100,000 genomes project. As well as wider integration across the Bioinformatics group software projects.</p>

<p><code class="highlighter-rouge">10.2018-04.2019</code>
<strong>Great Ormond Street Hospital</strong> Honorary bioinformatician role involving a collaboration to build on a bioinformatics management tool for handling clinical data retrieved from Genomics England as part of the 100,000 genomes project.</p>

<p><code class="highlighter-rouge">05.2018-present</code>
<strong>Imperial College London</strong> Honorary research fellow within the National Heart Lung Institute, providing bioinformatics support for several rare disease datasets generated as part of the 100,000 genomes project.</p>

<p><code class="highlighter-rouge">01.2018-04.2019</code>
<strong>Imperial College Healthcare NHS Trust</strong> Bioinformatics scientist working on the 100,000 Genomes Project, and part of the West London Genomic Medicine Centre (GMC).</p>

<p><code class="highlighter-rouge">04.2016-04.2018</code>
<strong>Faculty of Genomic Medicine, Queen Mary University London</strong> Visiting lecturer, responsible for delivering part of the HEE MSc in Genomic Medicine ‘Application of Genomics in Infectious Disease’ module.</p>

<p><code class="highlighter-rouge">07.2014-01.2018</code>
<strong>Public Health England</strong> Computational Biologist role, responsible for building and delivering genomic and metagenomic research and development projects using a range of NGS platforms.</p>

<p><code class="highlighter-rouge">09.2013-07.2014</code>
<strong>The Pirbright Institute</strong> Characterising terabase-scale sequence data from the transcriptome of Anopheles gambiae, as part of the Vector Molecular Biology group</p>

<p><code class="highlighter-rouge">02.2008-07.2009</code>
<strong>Wellcome Trust Sanger Institute</strong> Finishing and comparative analysis within the Pathogen Genomics group, including Clostridium difficile and Mycobacterium
tuberculosis.</p>

<h2 id="education">Education</h2>

<p><code class="highlighter-rouge">09.2009-06.2013</code>
<strong>MRC National Institute of Medical Research</strong> Pathogen Genomics PhD (supervisors: Prof. Douglas Young and Prof. Sebastein Gagneux). My thesis was entitled <em>A genomic and transcriptomic study of lineage-specific variation in Mycobacterium tuberculosis</em>, available via <a href="http://discovery.ucl.ac.uk/1397251/">UCL Senate House</a></p>

<p><code class="highlighter-rouge">09.2006-08.2007</code>
<strong>University of Leeds</strong> MRes Bioinformatics and Computational Biology (<em>Distinction</em>)</p>

<p><code class="highlighter-rouge">09.2003-06.2006</code>
<strong>University of Leeds</strong> BSc Microbiology (<em>Hons</em>) 2.1</p>

<h2 id="presentations">Presentations</h2>

<h3 id="national">National</h3>

<p><code class="highlighter-rouge">2017</code>
Microbiology Society event: Microbiome session at Porton Down, “Challenges of the Unknown”</p>

<p><code class="highlighter-rouge">2016</code>
Illumina User General Meeting: Invited scientific talk session, “The preterm infant gut microbiome”</p>

<h2 id="publications">Publications</h2>

<!-- ### Journals -->
<p><code class="highlighter-rouge">2021</code>
<strong>The Mycobacterium tuberculosis sRNA F6 regulates expression of groEL/S.</strong>
J Houghton, A Rodgers, <strong>G. Rose</strong>, KB Arnvig.. Microbiology spectrum. 2021</p>

<p><strong>Premature neonatal gut microbial community patterns supporting an epithelial TLR-mediated pathway for necrotizing enterocolitis</strong>
Shaw, A. G., Sim, K., <strong>Rose, G.</strong>, Wooldridge, D. J., Li, M. S., Misra, R. V., Gharbia, S., & Kroll, J. S. BMC microbiology. 2021<p>

<p><code class="highlighter-rouge">2020</code>
<strong>The airways microbiome of individuals with asthma treated with high and low doses of inhaled corticosteroids.</strong>
Martin, M. J., Zain, N. M. M., Hearson, G., Rivett, D. W., Koller, G., Wooldridge, D. J., <strong>Rose, G.</strong>, Gharbia, S. E., Forbes, B., Bruce, K. D., & Harrison, T. W. PloS one. 2020

<p><code class="highlighter-rouge">2017</code>
<strong>Complete Genome Sequences of BK Polyomavirus Strains from Two Patients with Urinary Tract Infection, Sequenced Using the Ion Torrent Platform.</strong>
<strong>Rose, G.</strong>, Ranellou, K., Misra, R., Crump, C., Wooldridge, D., Parmar, S., Maddren, C., Gharbia, S., & Jalal, H. Genome Announcements. 2017</p>

<p><strong>The Challenges of Identifying Mycobacterium to the Species Level using MALDI‐TOF MS</strong>
<strong>G. Rose</strong>, R. Culak, T. Chambers, S. E. Gharbia, H. N. Shah, Y. Kim, JS. Kim.  MALDI‐TOF and Tandem MS for Clinical Microbiology. 2017.</p>

<p><strong>Antibiotic resistance potential of the healthy preterm infant gut microbiome.</strong>
<strong>G. Rose</strong>, A. G. Shaw, K. Sim, D. J. Wooldridge, M. Li, S. Gharbia, R. Misra, J. S. Kroll. PeerJ. 2017.</p>

<p><code class="highlighter-rouge">2016</code>
<strong>Dosage compensation in the African malaria mosquito Anopheles gambiae.</strong>
<strong>G. Rose</strong>, E. Krzywinska, J. Kim, L. Revuelta, L. Ferretti, and J. Krzywinski. Genome Biology and Evolution. 2016.</p>

<p><strong>Development of a candidate reference material for adventitious virus detection in vaccine and biologicals manufacturing by deep sequencing</strong>. ET Mee, MD Preston, PD Minor, S Schepelmann; <strong>CS533 Study Participants</strong>. Vaccine. 2016.</p>

<p><code class="highlighter-rouge">2015</code>
<strong>Genomic diversity of BK polyomavirus in East of England–Preliminary data.</strong>
K Ranellou, C Crump, R Misra, D Wooldridge, <strong>G Rose</strong>, S Gharbia, S Parmar, M Curran, N Verlander, H Zhang, H Jalal. Journal of Clinical Virology. 2015.</p>

<p><strong>Challenges of the Unknown: Clinical Application of Microbial Metagenomics.</strong>
<strong>G. Rose</strong>, D. J. Wooldridge, C. Anscombe, E. T. Mee, R. V. Misra, and S. Gharbia. International Journal of Genomics. 2015.</p>

<p><strong>Complete genome sequence of the hypervirulent bacterium Clostridium difficile strain G46, ribotype 027.</strong>
T. Gaulton, R. Misra, <strong>G. Rose</strong>, P. Baybayan, R. Hall, J. Freeman, J. Turton, S. Picton, J. Korlach, S. Gharbia, and H. Shah. Genome Announcements. 2015.</p>

<p><code class="highlighter-rouge">2013</code>
<strong>A genomic and transcriptomic study of lineage-specific variation in Mycobacterium tuberculosis.</strong>
<strong>GD. Rose</strong>. University College London. 2013.</p>

<p><strong>Mapping of Genotype-Phenotype Diversity among Clinical Isolates of Mycobacterium tuberculosis by Sequence-Based Transcriptional Profiling.</strong>
<strong>G. Rose</strong>, T. Cortes, I. Comas, M. Coscolla, S. Gagneux, and D. B. Young. Genome Biology and Evolution. 2013.</p>

<p><strong>Genome-wide Mapping of Transcriptional Start Sites Defines an Extensive Leaderless Transcriptome in Mycobacterium tuberculosis.</strong>
T. Cortes, O. T. Schubert, <strong>G. Rose</strong>, K. B. Arnvig, I. Comas, R. Aebersold, and D. B. Young Cell Reports. 2013.</p>

<p><strong>A Small RNA Encoded in the Rv2660c Locus of Mycobacterium tuberculosis is Induced during Starvation and Infection.</strong>
J. Houghton, T. Cortes, O. Schubert, <strong>G. Rose</strong>, A. Rodgers, M. De Ste Croix, R. Aebersold, D. B. Young, and K. B. Arnvig. PLoS ONE. 2013.</p>

<p><strong>Use of whole genome sequencing to determine the microevolution of Mycobacterium tuberculosis during an outbreak.</strong>
M. Kato-Maeda, C. Ho, B. Passarelli, N. Banaei, J. Grinsdale, L. Flores, J. Anderson, M. Murray, <strong>G. Rose</strong>, L. M. Kawamura, N. Pourmand, M. a. Tariq, S. Gagneux, and P. C. Hopewell. PLoS One. 2013.</p>

<p><code class="highlighter-rouge">2012</code>
<strong>The heterogeneous evolution of multidrug-resistant Mycobacterium tuberculosis.</strong>
B. Müller, S. Borrell, <strong>G. Rose</strong>, and S. Gagneux. Trends Genet. 2012.</p>

<p><code class="highlighter-rouge">2011</code>
<strong>Whole-genome sequencing of rifampicin-resistant Mycobacterium tuberculosis strains identifies compensatory mutations in RNA polymerase genes.</strong>
I. Comas, S. Borrell, A. Roetzer, <strong>G. Rose</strong>, B. Malla, M. Kato-Maeda, J. Galagan, S. Niemann, and S. Gagneux. Nature Genetics 2011.</p>

<p><strong>Sequence-based analysis uncovers an abundance of non-coding RNA in the total transcriptome of Mycobacterium tuberculosis.</strong>
K. B. Arnvig, I. Comas, N. R. Thomson, J. Houghton, H. I. Boshoff, N. J. Croucher, <strong>G. Rose</strong>, T. T. Perkins, J. Parkhill, G. Dougan, and D. B. Young. PLoS Pathogens. 2011.</p>

<p><strong>Genotypic and phenotypic modifications of Neisseria meningitidis after an accidental human passage.</strong>
H. Omer, <strong>G. Rose</strong>, K. a. Jolley, E. Frapy, J. R. Zahar, M. C. J. Maiden, S. D. Bentley, C. R. Tinsley, X. Nassif, and E. Bille. PLoS One. 2011.</p>

<p><code class="highlighter-rouge">2010</code>
<strong>Evolutionary dynamics of Clostridium difficile over short and long time scales.</strong>
M. He, M. Sebaihia, T. D. Lawley, R. A. Stabler, L. F. Dawson, M. J. Martin, K. E. Holt, H. M. B. Seth-Smith, M. a Quail, R. Rance, K. Brooks, C. Churcher, D. Harris, S. D. Bentley, C. Burrows, L. Clark, C. Corton, V. Murray, <strong>G. Rose</strong>, S. Thurston, A. van Tonder, D. Walker, B. W. Wren, G. Dougan, and J. Parkhill. PNAS. 2010.</p>

<p><code class="highlighter-rouge">2009</code>
<strong>Comparative genome and phenotypic analysis of Clostridium difficile 027 strains provides insight into the evolution of a hypervirulent bacterium.</strong>
R. A. Stabler, M. He, L. Dawson, M. Martin, E. Valiente, C. Corton, T. D. Lawley, M. Sebaihia, M. A. Quail, <strong>G. Rose</strong>, D. N. Gerding, M. Gibert, M. R. Popoff, J. Parkhill, G. Dougan, and B. W. Wren. Genome Biology. 2009.</p>

<h2 id="technical-skills">Technical skills</h2>

<ul>
  <li>Linux sysadmin</li>
  <li>Shell scripting - bash/awk/sed</li>
  <li>Python - pandas, beautifulsoup, RESTful APIs, microservices</li>
  <li>Full Stack development - Django, JavaScript, Docker, PostgreSQL, and asynchronous tasks with Celery</li>
  <li>HPC - batching, lustre</li>
  <li>Java/Groovy - pipelining with bpipe, snakemake and Nextflow workflows</li>
  <li>Perl - general scripting</li>
  <li>VBA - lifting legacy code into Python</li>
  <li>R - bioconductor, DESeq</li>
  <li>GitLab DevOps, setting up maintaining CI/CD pipelines</li>
</ul>

<h2 id="areas-of-expertise">Areas of expertise</h2>
<ul>
  <li>Bioinformatics</li>
  <li>Analysis pipeline development</li>
  <li>NGS and variant calling</li>
  <li>Bacterial genomics</li>
</ul>

<h2 id="awards">Awards</h2>

<p>Public Health England Pump Prime Funding (PPF) round 2017 (viral surveillance study)</p>

<p>MRC scholarship (PhD fees + stipend)</p>

<p>BBSRC postgraduate scholarship (MRes fees + stipend)</p>

<h2 id="links">Links</h2>

<!-- fa are fontawesome, ai are academicons -->
<ul>
  <li><i class="fa fa-envelope"></i> <a href="mailto:grahamdrose@gmail.com">grahamdrose</a><br /></li>
  <li><i class="fa fa-github"></i> <a href="http://github.com/byronmews">github</a><br /></li>
  <li><i class="fa fa-linkedin"></i> <a href="https://www.linkedin.com/in/graham-rose-033a9829/">LinkedIn</a></li>
  <li><i class="ai ai-google-scholar"></i> <a href="http://scholar.google.com/citations?user=TIkDO_cAAAAJ&amp;hl=en">Google scholar</a></li>
  <li><i class="ai ai-orcid"></i> <a href="https://orcid.org/0000-0002-5097-5426">ORCID</a></li>
</ul>

<!-- ### Footer

Last updated: Sep 2018 -->

    </div>
  </div>

  <div class="footer"></div>

</body>
</html>
